manifest/package.go

```diff
@@ -119,12 +119,41 @@ func (r *manifestConverter) manifestToPkg(manif *Manifest) (*pbsubstreams.Packag
 	return pkg, protoDefinitions, r.sinkConfigDynamicMessage, nil
 }
 
+func (m *Manifest) readFileFromName(filename string) ([]byte, error) {
+	if _, err := os.Stat(filepath.Join(m.Workdir, filename)); err == nil {
+		filePath := filepath.Join(m.Workdir, filename)
+		content, err := os.ReadFile(filePath)
+		if err != nil {
+			return nil, fmt.Errorf("failed to read %s: %w", filename, err)
+		}
+		return content, nil
+	}
+	return nil, nil
+}
+
 func (r *manifestConverter) convertToPkg(m *Manifest) (pkg *pbsubstreams.Package, err error) {
+	doc := m.Package.Doc
+	if doc == "" {
+		readmeContent, err := m.readFileFromName("README.md")
```

Comment:
you'll have a NotFound error here, you need to check fall back from there for checking README..